### ISSUE TITLE
Menu and cookies?

### ISSUE THREAD

**Issue by StopAD** [emilianobovetti/FuckFuckAdBlock#11]
Sometimes I don't see the icons colored; seem it is related to cookies on off.

![624](https://cloud.githubusercontent.com/assets/20204871/16449527/4153355e-3df8-11e6-81ca-5acdd023bd48.jpg)

**Comment by emilianobovetti** [comment]
You're right, the current mode was saved on `localStorage`, by the way I will probably drop this project so excuse me if I won't fix this.
